ShopApp/App/Modules/Account/ViewModels/PersonalInfoViewModel.swift

```diff
@@ -10,10 +10,9 @@ import RxSwift
 import ShopApp_Gateway
 
 class PersonalInfoViewModel: BaseViewModel {
-    private let loginUseCase = LoginUseCase()
-    private let customerUseCase = CustomerUseCase()
-    
     private var updateCustomerUseCase: UpdateCustomerUseCase
+    private let loginUseCase: LoginUseCase
```

Comment:
`private let` variables should be before `private var` and separate with one line. It should be in all classes.

Reply:
Also, all use cases should be `let` or `var`. We can't have one `var` use case and `let` all other use cases.